src/components/Datepicker/Datepicker.js

```diff
@@ -25,6 +28,14 @@ function Datepicker({
   const [prevDates, setPrevDates] = useState({})
   const [deepNavVisible, setDeepNavVisibility] = useState(false)
 
+  useEffect(() => {
+    if (datePickerRef && datePickerRef.current) {
+      innerRef(datePickerRef.current)
+    }
+    // No need to add innerRef prop to the deps list
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [datePickerRef])
```

Comment:
I think that `datePickerRef.current` would trigger the useEffect callback correctly here (not 100% sure though, maybe just `datePickerRef` is fine)